.github/workflows/test.yml

```diff
@@ -46,14 +46,9 @@ jobs:
 #          echo Add other actions to build,
 #          echo test, and deploy your project.
 
-      - name: Create Javadocs
-        run: |
-          cd newrelic-java-agent/newrelic-api 
-          ../gradlew javadoc -Prelease=true -PagentVersion=8.0.0
-          ls build/docs/javadoc
-          
       - name: set gradle.properties
         run: |
+          cd ${GITHUB_WORKSPACE}/newrelic-java-agent
```

Comment:
I had to play a little with the current directory.
So to make it easier to navigate, I added this ${GITHUB_WORKSPACE} which is the base place where things are checked out to.